quinn-udp/src/unix.rs

```diff
@@ -712,109 +712,132 @@ fn prepare_recv(
     hdr.msg_datalen = buf.len();
 }
 
+/// Decoded metadata from control messages.
+struct DecodedCmsg {
```

Comment:
I find this name confusing because it represents a number of control messages. Maybe `ControlMetadata` or something?